README.adoc

```diff
@@ -6,26 +6,29 @@ The metrics collector is a companion application to the http://cloud.spring.io/s
-|metrics-collector-kafka-09| http://repo.spring.io/libs-release/org/springframework/cloud/metrics-collector-kafka-09/1.0.0.RELEASE/metrics-collector-kafka-09-1.0.0.RELEASE.jar | https://hub.docker.com/r/springcloud/metrics-collector-kafka-09/tags/ [docker pull metrics-collector-kafka-09:1.0.0.RELEASE]
-|metrics-collector-kafka-10| http://repo.spring.io/libs-release/org/springframework/cloud/metrics-collector-kafka-10/1.0.0.RELEASE/metrics-collector-kafka-10-1.0.0.RELEASE.jar | https://hub.docker.com/r/springcloud/metrics-collector-kafka-10/tags/ [docker pull metrics-collector-kafka-10:1.0.0.RELEASE]
+|metrics-collector-rabbit| http://repo.spring.io/libs-release/org/springframework/cloud/metrics-collector-rabbit/2.0.0.RELEASE/metrics-collector-rabbit-2.0.0.RELEASE.jar| https://hub.docker.com/r/springcloud/metrics-collector-rabbit/tags/ [docker pull metrics-collector-rabbit:2.0.0.RELEASE]
+|metrics-collector-kafka| http://repo.spring.io/libs-release/org/springframework/cloud/metrics-collector-kafka/2.0.0.RELEASE/metrics-collector-kafka-2.0.0.RELEASE.jar | https://hub.docker.com/r/springcloud/metrics-collector-kafka/tags/ [docker pull metrics-collector-kafka:2.0.0.RELEASE]
 |======================
 
 ## Downloadable Links for Latest Snapshots for Metrics Collector Uber Jars and Docker Images
 
 [width="40%",frame="topbot",options="header,footer"]
 |======================
 |Uber Jar |Http Link |Docker Hub Link
-|metrics-collector-rabbit| http://repo.spring.io/libs-snapshot-local/org/springframework/cloud/metrics-collector-rabbit/1.0.1.BUILD-SNAPSHOT/metrics-collector-rabbit-1.0.1.BUILD-SNAPSHOT.jar| https://hub.docker.com/r/springcloud/metrics-collector-rabbit/tags/ [docker pull metrics-collector-rabbit:1.0.1.BUILD-SNAPSHOT]
-|metrics-collector-kafka-09| http://repo.spring.io/libs-snapshot-local/org/springframework/cloud/metrics-collector-kafka-09/1.0.1.BUILD-SNAPSHOT/metrics-collector-kafka-09-1.0.1.BUILD-SNAPSHOT.jar | https://hub.docker.com/r/springcloud/metrics-collector-kafka-09/tags/ [docker pull metrics-collector-kafka-09:1.0.1.BUILD-SNAPSHOT]
-|metrics-collector-kafka-10| http://repo.spring.io/libs-snapshot-local/org/springframework/cloud/metrics-collector-kafka-10/1.0.1.BUILD-SNAPSHOT/metrics-collector-kafka-10-1.0.1.BUILD-SNAPSHOT.jar | https://hub.docker.com/r/springcloud/metrics-collector-kafka-10/tags/ [docker pull metrics-collector-kafka-10:1.0.1.BUILD-SNAPSHOT]
+|metrics-collector-rabbit| http://repo.spring.io/libs-snapshot-local/org/springframework/cloud/metrics-collector-rabbit/2.0.1.BUILD-SNAPSHOT/metrics-collector-rabbit-2.0.1.BUILD-SNAPSHOT.jar| https://hub.docker.com/r/springcloud/metrics-collector-rabbit/tags/ [docker pull metrics-collector-rabbit:2.0.1.BUILD-SNAPSHOT]
```

Comment:
Same. We haven't released 2.0 yet, and `2.0.1.BUILD-SNAPSHOT` is a stretch.

Reply:
Can reference 2.0.0.BUILD-SNAPSHOT